### DISCUSSION TITLE
Creation of a fast Ripes executable

### DISCUSSION THREAD

**Discussion by Y1annis** [mortbopet/Ripes#271] [General]
Hello, I have noticed that my Ripes executable (By compiling using Qt) is much slower than the published executable in github. I wanted to ask if you did something during the creation of the executable that made it so much faster.

Thank you for your time.

**Comment by mortbopet** [comment]
Are you building in Debug mode? The published executables are built in Release mode (see https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html).

**Reply by Y1annis** [reply]
Thank you so much for the fast reply. Yes I am building in debug mode

**Reply by Y1annis** [reply]
Thank you so much, it is much faster now
